Quiver/Validating/Default Validators/CustomValidator.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CustomValidator.swift
+//  Quiver
+//
+//  Created by Luciano Almeida on 27/09/17.
+//  Copyright © 2017 Heitor Costa. All rights reserved.
+//
+
+import Foundation
+
+class CustomValidator: Validator {
+    
+    typealias ValidationClousure = (Any?) -> Bool
+    
+    var closure: ValidationClousure
+    
+    init(closure: @escaping ValidationClousure) {
+        self.closure = closure
+    }
+    
+    override func validate(with object: Any?) throws -> Bool {
```

Comment:
If you see the other validators, they return **true** with **nil** value, since this is a validation that the `RequiredValidator` takes care of. Should we ignore it in this one, leaving the programmer to define this inside the closure or should we infer this and return true before the closure even begins?

Reply:
Ignore makes more sense to me, because I think if in this case the closure is not called it can be consider by the user as a bug(the closure to validate it's not being called). Unless if it's explicit to the user that the closure will not be called on nil values

Reply:
Yeah, that makes sense. If infer is needed, we can always change it later.